### ISSUE TITLE
Check if rtx bin path is already on PATH when calling rtx activate

### ISSUE THREAD

**Issue by amoosbr** [jdxcode/rtx#10]
When `eval "$(rtx activate -s zsh)"` is run on shell loading, the directory of the rtx binary is added to the front of PATH.
Although the directory is already on the path. Therefore overwriting my custom PATH order.

Fix:

rtx checks, if the bin path is already in PATH and only updates the PATH when needed.

Reproduce:

`.zshrc`
```sh
echo $PATH
eval "$(rtx activate -s zsh)"
echo $PATH
```

start new terminal shell:
```sh
/Users/user/bin:/Users/user/.local/bin:/Users/user/.cargo/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin

# /opt/homebrew/bin: added although already on the PATH, overriding the custom order

/opt/homebrew/bin:/Users/user/bin:/Users/user/.local/bin:/Users/user/.cargo/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin

```

**Comment by jdxcode** [comment]
shoot, I actually thought of this and checked to make sure it behaved correctly for fish but didn't think about bash/zsh. This should be an easy fix.

**Comment by jdxcode** [comment]
this fix is out in 1.2
